AppStoreExample/AppStoreExample/ViewModels/AppSyncViewModel.swift

```diff
@@ -8,43 +8,51 @@
 
 import Foundation
 
+enum AppSyncActions {
+    case idle
+    case dataWasFetched(error: Error?)
+    case fetchingData
```

Comment:
I would rename dataWasFetched by 'completed' or 'finished' the word 'fetch" i think adds confusion since it is already a sync operation.